frontend/src/store/FamLoginUserState.ts

```diff
@@ -257,6 +259,9 @@ const cacheUserAccess = async () => {
     }
 };
 
+// todo: check if login user needs to accept terms and conditions, if need, show terms here
+const requiresAcceptTermsCondition = async () => {};
```

Comment:
Hi Jota, this will be place where we call the backend api to decide if show terms or not.

Reply:
Yes, this should be the place to check.